config/routes.rb

```diff
@@ -1,22 +1,22 @@
 Vimgolf::Application.routes.draw do
 
-  match "/auth/twitter/callback" => "sessions#create"
-  match "/signout" => "sessions#destroy", :as => :signout
+  match "/auth/twitter/callback" => "sessions#create", via: [:get, :post]
```

Comment:
yeah sorry, did that initially to make sure we were letting through all the requests we needed for specs. now that everything is passing, i can go through and update the route syntax 👍